### PR TITLE
feat/VEGA-5568 made store, epics, selectors for configuration

### PR DESCRIPTION
**Pull request by SergeyKatugin**
## Задача
Задача в Jira: https://jira.konakov.tv/browse/VEGA-5568
## Problem statement
https://artcpt.atlassian.net/browse/VEGA-5568
https://artcpt.atlassian.net/browse/VEGA-5569

## Solution details
Было реализовано:
* Интерфейсы полей ввода и метаданных
* Хранение их в сторе
* Action, Reducers для них же
* Добавлены эпики для запроса данных
* Добавлен обработчик на "конфигурировать", чтоб запрашивать все данные
* Добавлены различные селекторы, которые помогут вытащить нужные вам данные, в зависимости от активного таба/метаданных

## Type

- [x] Feature
- [ ] Bug Fix
- [x] Refactoring
- [ ] Test
- [ ] Other

## Quality control

- [x] PR: Based on a correct branch
- [x] PR: Head branch is rebased on the base branch
- [x] PR: Assignee chosen and necessary labels are set
- [x] PR: Current form is filled out (where it makes sense)
- [x] PR: Diff checked for irrelevant changes
- [x] PR: Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specs
- [x] PR: Checked for spelling, grammar and typos
- [x] JS: There's no errors/warnings in the browser dev console
- [x] Layout: Tested with various content amounts
- [ ] Docs: All the important changes and features are described
- [ ] Tests: Configuration files are checked at test repositories
- [ ] Tests: New unit tests developed
- [ ] Tests: New E2E tests developed
- [ ] Tests: Existing unit tests passed successfully
- [ ] Tests: Existing E2E tests passed successfully
- [ ] Tests: Manually tested on personal instance

## Notable statements
- [ ] Affects configuration files
- [ ] Brings new packages or updates existing
- [ ] Opened follow-up tasks to resolve

### CODE REVIEW

File: src/store/canvasMetadata/CanvasMetadataEpics.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ * Логика в следующем:
+ * Мы где то вызываем action handleInitMetadataByComponentId
+ * В котором передаем component id, по которому хотим загрузить метаданные
+ * Далее эпик handleInitMetadataByComponentId это отлавливает, устанавливает лоадер
+ * И отправляет в websocket
+ *
+ * Далее эпик handleGetMetadataDictionary ловит получение из сокета
+ * Добавляет в стору
+ * И далее нам нужно получить все возможные инпуты данной метадаты
+ * Для этого нам нужно специальным методом(getInputIdsByMetadata) найти из все
```
Comment (ndrzhn) [inline src/store/canvasMetadata/CanvasMetadataEpics.ts:24]: их все* - опечатка, видимо

**Review by ndrzhn — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,43 +1,50 @@
...
+- [ ] PR: Выбран Assignee и проставлены нужные лейблы
+- [ ] PR: Данная форма заполнена (пункты, которые имеют значение)
+- [ ] PR: Дифф проверен на неактуальные изменения
+- [ ] PR: Сообщения коммитов написаны по правилам [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
+- [ ] PR: Проверен на орфографию, грамматику и опечатки
+- [ ] JS: Нет errors/warnings в консоли браузера
+- [ ] Layout: Протестирован с различным содержимым
+- [ ] Docs: Все важные измененния и фичи описаны
+- [ ] Tests: Конфигурационные файлы проверяются в тестовых репозиториях
+- [ ] Tests: Разаработаны новые unit тесты
```
Comment (satrasmatras) [inline .github/pull_request_template.md:41]: Опечатка

**Review by satrasmatras — APPROVED** [review]

File: src/store/canvasMetadata/CanvasMetadataEpics.ts (hunk 1 above)
Comment (SergeyKatugin) [reply src/store/canvasMetadata/CanvasMetadataEpics.ts:24]: ок

File: .github/pull_request_template.md (hunk 2 above)
Comment (SergeyKatugin) [reply .github/pull_request_template.md]: ok

**Pull request merged**